manifest.json

```diff
@@ -7,8 +7,9 @@
   "permissions": [
     "tabs",
     "storage",
-    "https://www.facebook.com",
-    "https://www.linkedin.com"
+    "https://www.facebook.com/",
+    "https://www.linkedin.com/",
+    "https://www.twitter.com/"
```

Comment:
Hi @fortunatodeangelis !
Just one little question: I see that both Facebook and LinkedIn redirects their users to a `www` domain, but Twitter doesn't, it just uses `twitter.com` as main domain. Will this permission request work on Twitter?